ground-station-openmct/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "openmct-tutorials",
```

Comment:
Can you rename this/remove the references to the tutorial? I think you can just get rid of most of these sections